instance/static/html/instance/appserver.html

```diff
@@ -214,7 +214,7 @@ <h6 ng-if="appserver.status == 'failed'">Full Log</h6>
            ng-repeat="line in appserverLogs.log_entries track by $index">
         <span class="timestamp">{{ line.created | date:'yyyy-MM-dd HH:mm:ssZ' }}</span>
         <span class="log-level">{{ line.level }}</span>
-        <pre>{{ line.text | stripLogMeta }}</pre>
+        <pre>{{ line.text | stripLogMeta | prettifyJSON }}</pre>
```

Comment:
@mtyaka Would be great if we also had a filter to remove the `stdout` and `stderr` keys which are both unreadable and duplicated by the `stdout_lines` and `stderr_lines` keys. Maybe for a followup ticket?

Reply:
I didn't notice `stdout`/`stdout_lines` and `stderr`/`stderr_lines` is always duplicated, nice spot! I'll modify the filter to remove `stdout`/`stderr` from the output if the corresponding `*_lines` entry is present.